packages/go/cypher/backend/pgsql/format_test.go

```diff
@@ -0,0 +1,714 @@
+			ExpectedParameters: map[string]interface{}{
+				"p0":  "67CE0FEC-166C-4E5E-BF87-6FBAF0E9C8A8",
+				"p1":  "CLIENTAUTH@ESC1.LOCAL",
+				"p10": false,
+				"p11": "SUBJECT_ALT_REQUIRE_UPN, SUBJECT_REQUIRE_DIRECTORY_PATH",
+				"p12": false,
+				"p13": true,
+				"p14": MustMarshalToJSONB([]string{"1.3.6.1.5.5.7.3.2"}),
+				"p15": MustMarshalToJSONB([]string{}),
+				"p16": 0,
+				"p17": MustMarshalToJSONB([]string{}),
+				"p18": MustMarshalToJSONB([]string{}),
+				"p19": MustMarshalToJSONB([]string{"1.3.6.1.5.5.7.3.2"}),
+				"p2":  "S-1-5-21-909015691-3030120388-2582151266",
+				"p20": true,
+				"p3":  "CN=CLIENTAUTH,CN=CERTIFICATE TEMPLATES,CN=PUBLIC KEY SERVICES,CN=SERVICES,CN=CONFIGURATION,DC=ESC1,DC=LOCAL",
+				"p4":  "1 year",
+				"p5":  "6 weeks",
+				"p6":  1,
+				"p7":  "1.3.6.1.4.1.311.21.8.12059088.7148202.5130407.12905872.6174753.77.1.4",
+				"p8":  "AUTO_ENROLLMENT",
+				"p9":  false},
+		},
+
+		// UPDATE CASES
+		{
+			ID:       158,
+			Source:   "match (s) where s:NodeKindA set s:NodeKindB return s",
+			Expected: "update node as s set kind_ids = kind_ids || @p0 where s.kind_ids operator(pg_catalog.&&) array[1]::int2[] returning (s.id, s.kind_ids, s.properties)::nodeComposite as s",
+		},
+		{
+			ID:       159,
+			Source:   "match (s) where s:NodeKindA set s:NodeKindB remove s:NodeKindA return s",
+			Expected: "update node as s set kind_ids = kind_ids - @p1 || @p0 where s.kind_ids operator(pg_catalog.&&) array[1]::int2[] returning (s.id, s.kind_ids, s.properties)::nodeComposite as s",
+		},
+		{
+			ID:       160,
+			Source:   "match (s) set s.name = 'new name', s:NodeKindA return s",
+			Expected: "update node as s set properties = properties || @p0, kind_ids = kind_ids || @p1 returning (s.id, s.kind_ids, s.properties)::nodeComposite as s",
+		},
+		{
+			ID:       161,
+			Source:   "match (s) where s:NodeKindA set s.name = 'new name' return s",
+			Expected: "update node as s set properties = properties || @p0 where s.kind_ids operator(pg_catalog.&&) array[1]::int2[] returning (s.id, s.kind_ids, s.properties)::nodeComposite as s",
+		},
+		{
+			ID:       162,
+			Source:   "match (s) where s:NodeKindA set s.name = 'lol' remove s.other return s",
+			Expected: "update node as s set properties = properties - @p1::text[] || @p0 where s.kind_ids operator(pg_catalog.&&) array[1]::int2[] returning (s.id, s.kind_ids, s.properties)::nodeComposite as s",
+		},
+		//{
+		//	ID:        63,
+		//	Source:    "match p = allShortestPaths((:NodeKindA)-[:EdgeKindA*..]->(:NodeKindB)) return p",
+		//	Expected:  "",
+		//	Exclusive: true,
+		//},
```

Comment:
please remove dead code or document why this is commented out

Reply:
Patched up